tests/utils/notebook_test/__init__.py

```diff
@@ -31,7 +31,7 @@
 
 
 IPYTHON_VERSION = 4  # Pin to ipython version 4.
-TIME_OUT = 10*60  # Maximum 10 mins/test. Reaching timeout causes test failure.
+TIME_OUT = 15*60  # Maximum 10 mins/test. Reaching timeout causes test failure.
```

Comment:
nit: can you update the comment to max 15 mins/ test

Reply:
i am going to merge this PR to unblock nightlies. @access2rohit since this change is small, would you be able to add this change to one of your WIP PRs?